homeassistant/components/roomba/strings.json

```diff
@@ -0,0 +1,32 @@
+{
+  "config": {
+    "title": "iRobot Roomba",
+    "step": {
+      "user": {
+        "title": "Connect to the device",
```

Comment:
"description": "Currently retrieving the BLID and password is a manual process.  Please follow the steps outlined in the documentation at: https://www.home-assistant.io/integrations/roomba/#retrieving-your-credentials"